src/panels/lovelace/cards/hui-entities-card.ts

```diff
@@ -0,0 +1,195 @@
+import { html, LitElement, PropertyDeclarations } from "@polymer/lit-element";
+
+import "../../../components/ha-card.js";
+import "../components/hui-entities-toggle.js";
+
+import { fireEvent } from "../../../common/dom/fire_event.js";
+import { DOMAINS_HIDE_MORE_INFO } from "../../../common/const.js";
+import { HassLocalizeLitMixin } from "../../../mixins/lit-localize-mixin";
+import { LovelaceCard, LovelaceConfig } from "../types.js";
+import createRowElement from "../common/create-row-element.js";
+import computeDomain from "../../../common/entity/compute_domain.js";
+import processConfigEntities from "../common/process-config-entities";
+import { HomeAssistant } from "../../../types.js";
+
+interface EntityConfig {
+  name?: string;
+  icon?: string;
+  entity: string;
+  type?: string;
+  secondary_info: "entity-id" | "last-changed";
+  action_name?: string;
+  service?: string;
+  service_data?: object;
+  url?: string;
+}
+
+interface Config extends LovelaceConfig {
+  show_header_toggle?: boolean;
+  title?: string;
+  entities: EntityConfig[];
+}
+
+class HuiEntitiesCard extends HassLocalizeLitMixin(LitElement)
+  implements LovelaceCard {
+  protected _hass?: HomeAssistant;
+  protected config?: Config;
+  protected configEntities?: EntityConfig[];
+
+  set hass(hass) {
+    this._hass = hass;
+    if (this.shadowRoot && this.shadowRoot.querySelector("ha-card")) {
+      this.shadowRoot
+        .querySelector("ha-card")!
+        .querySelectorAll("#states > div > *")
```

Comment:
why not query for this right away? No reason to use `ha-card` as inbetween. 

And why are our entity rows wrapped in a div?

Reply:
They are wrapped in divs in the JS version but without the wrapping the spacing is off. But can be fixed with css